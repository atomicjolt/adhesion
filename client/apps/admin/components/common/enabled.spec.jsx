import React from 'react';
import { shallow } from 'enzyme';
import Enabled from './enabled';

describe('the enabled component', () => {
  it('matches the snapshot', () => {
    const result = shallow(<Enabled />);
    expect(result).toMatchSnapshot();
  });
});
