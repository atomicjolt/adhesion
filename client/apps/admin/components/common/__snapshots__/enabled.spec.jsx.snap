// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the enabled component matches the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Enabled />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "check_circle",
      "className": "material-icons enabled",
    },
    "ref": null,
    "rendered": "check_circle",
    "type": "i",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "check_circle",
        "className": "material-icons enabled",
      },
      "ref": null,
      "rendered": "check_circle",
      "type": "i",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
